package.json

```
@@ -43,6 +43,13 @@
     "typescript": "4.1.3"
   },
   "peerDependencies": {
-    "eslint": "^7.5.0"
+    "eslint": "^7.5.0",
```

Should we pin peer dependencies to avoid any surprises? 🤔 
```suggestion
    "eslint": "7.5.0",
```

---

We had issues with this before. When the version was fixed to 6.8 (or something like that). 

This would cause issues in applications as they would be forced to use a specific version. On our project it would mean that we would need to combine all eslint renovate tasks into 1 to satisfy the rules.

It would also become a blocker for the applications where this lib would need to be updated before any potential security updates.

---

Yeah, I thought so. Thanks